appveyor.yml

```diff
@@ -207,7 +207,9 @@ test_script:
   - sh src/ci/init_repo.sh . /c/cache/rustsrc
   - set SRC=.
   - set NO_CCACHE=1
-  - sh src/ci/run.sh
+  # Try specifically using git's `sh.exe` instead of the one in PATH which is
+  # the msys64 sh.exe. Trying to debug rust-lang/rust#58160
+  - C:\Program Files\Git\usr\bin\sh.exe src/ci/run.sh
```

Comment:
Not really familiar with the windows shell, but shouldn't "Program Files" be escaped?

Reply:
Honestly I'm not sure. I figured we could try this and see what happens, but I think you're right